blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -33,9 +32,8 @@ export function calculateNodeBaseUrl(config) {
 
 export function calculateStepsBaseUrl(config) {
     const { name, runId, branch, _appURLBase, isMultiBranch, node } = config;
-    let baseUrl =
-        `${_appURLBase}/rest/organizations/jenkins/` +
-        `pipelines/${uriString(name)}`;
+    // TODO: can't hard-code org
+    let baseUrl = `${_appURLBase}/rest/organizations/jenkins/pipelines/${name}`;
```

Comment:
Maybe should open a ticket speciifcally for this? as it will bite (grepping for /jenkins/ shouldn't be too hard...